File: server/routes/ask_gpt.py
from flask import Blueprint, request
from controllers import gpt_controller


ask_gpt_routes = Blueprint('ask-gpt', __name__)


@ask_gpt_routes.route('/', methods=['GET'])
def authenticate():
    question = request.args.get('question')
    return gpt_controller.ask_gpt(question)

File: client/src/api/askGpt.js
import { HOST_IP } from './index';
import { request } from './request';

const ENDPOINTS = {
  askGpt: (text) => (`${HOST_IP}/ask-gpt?question=${text}`)
};

export const askGpt = (text) => {
  return request(
    ENDPOINTS.askGpt(text),
    { method: 'GET' }
  );
}